x-pack/plugins/security_solution/public/flyout/document_details/shared/context.tsx

```diff
@@ -75,7 +79,7 @@ export type DocumentDetailsProviderProps = {
 } & Partial<DocumentDetailsProps['params']>;
 
 export const DocumentDetailsProvider = memo(
```

Comment:
I know this PR doesn't really deal with that, but would you be ok adding a small description for this? Something like `Returns a context provider to be used for all panels of the document details expandable flyout`, or something better than that 😆